File: DataGenerator/src/datagenerator/WFServer.java
package datagenerator;

/**
 *
 * @author bilorge
 */
public class WFServer {
//  #this class is a representation of basic server implementation in a Wells Fargo data center model
    private String fqn; //   # ( K ) - Defines the server's fully qualified name
    private String name; //  # ( L ) - Defines the server's name
    private String ip; //    # ( M ) - Defines the server's IP
    private String os; //    # is it provided by column I or should be derived from column O?
    private String osv; //   # ( O ) - Defines the server's OS Version
    private String env; //   # ( E ) - Defines the server's environment
    private String dtc; //   # ( U ) - Defines the data center where the server resides

    public WFServer() {
    }

    public WFServer(String fqn, String name, String ip, String os, String osv, String env, String dtc) {
        this.fqn = fqn;
        this.name = name;
        this.ip = ip;
        this.os = os;
        this.osv = osv;
        this.env = env;
        this.dtc = dtc;
    }

    @Override
    public String toString() {
        return "WFServer{" + "fqn=" + fqn + ", name=" + name + ", ip=" + ip + ", os=" + os + ", osv=" + osv + ", env=" + env + ", dtc=" + dtc + '}';
    }
    
    /**
     * @return the fqn
     */
    public String getFqn() {
        return fqn;
    }

    /**
     * @param fqn the fqn to set
     */
    public void setFqn(String fqn) {
        this.fqn = fqn;
    }

    /**
     * @return the name
     */
    public String getName() {
        return name;
    }

    /**
     * @param name the name to set
     */
    public void setName(String name) {
        this.name = name;
    }

    /**
     * @return the ip
     */
    public String getIp() {
        return ip;
    }

    /**
     * @param ip the ip to set
     */
    public void setIp(String ip) {
        this.ip = ip;
    }

    /**
     * @return the os
     */
    public String getOs() {
        return os;
    }

    /**
     * @param os the os to set
     */
    public void setOs(String os) {
        this.os = os;
    }

    /**
     * @return the osv
     */
    public String getOsv() {
        return osv;
    }

    /**
     * @param osv the osv to set
     */
    public void setOsv(String osv) {
        this.osv = osv;
    }

    /**
     * @return the env
     */
    public String getEnv() {
        return env;
    }

    /**
     * @param env the env to set
     */
    public void setEnv(String env) {
        this.env = env;
    }

    /**
     * @return the dtc
     */
    public String getDtc() {
        return dtc;
    }

    /**
     * @param dtc the dtc to set
     */
    public void setDtc(String dtc) {
        this.dtc = dtc;
    }
    
}

File: python/wfserver.py
#using the following data dump to model the object
# COLUMN    VALUE                   COMMENT
# ( A 0  )  CI_NAME                 Sever name?
# ( B 1  )  CI_ID                   
# ( C 2  )  STATUS                  
# ( D 3  )  STATUSREASON            
# ( E 4  )  SYSTEMROLE              Possibly identifies to which environment server belongs to?
# ( F 5  )  SERVER_FUNCTION         
# ( G 6  )  OWNED_BY                
# ( H 7  )  APP_SUPPORT             
# ( I 8  )  OS_SUPPORT              determines the OS host uses?
# ( J 9  )  SA_DVC_ID               
# ( K 10 )  SA_SERVER NAME          Provides the FQN server name.
# ( L 11 )  SA_LOCALHOSTNAME        Provide the server name
# ( M 12 )  SA_PRIMARY_IP           
# ( N 13 )  SA_MANAGEMENT_IP        
# ( O 14 )  SA_OS_VERSION           Provise the sever's OS version
# ( P 15 )  SA_AGENT_VERSION        
# ( Q 16 )  SA_AGENT_INTIAL_REG     
# ( R 17 )  SA_UNREACHABLEDAYS      
# ( S 18 )  STATE                   
# ( T 19 )  SA_CUSTOMER             
# ( U 20 )  SA_FACILITY             This represents the data center?
# ( V 21 )  PRIMARYCORE             
# ( W 22 )  SA_MESH                 
# ( X 23 )  CERTOWNER               
# ( Y 24 )  CERTVALIDDATE           

#this class is a representation of basic server implementation in a Wells Fargo data center model
fqn = ''    # ( K ) - Defines the server's fully qualified name
name = ''   # ( L ) - Defines the server's name
ip = ''     # ( M ) - Defines the server's IP
os = ''     # is it provided by column I or should be derived from column O?
osv = ''    # ( O ) - Defines the server's OS Version
env = ''    # ( E ) - Defines the server's environment
dtc = ''    # ( U ) - Defines the data center where the server resides
